### DISCUSSION TITLE
Android showing empty popup when adding new payment method

### DISCUSSION THREAD

**Discussion by truong15** [hyochan/react-native-iap#2167] [General]
iOS is working well but for android, on beta and store versions, the UI is strange.

When there is no payment method yet, if the user chooses a payment method, an empty popup shows instead of the credit card info input form.

![methods](https://user-images.githubusercontent.com/10149093/208573433-9fa0a917-8fc3-442a-9d2c-e7f340e28758.jpeg)

![empty](https://user-images.githubusercontent.com/10149093/208573446-1479c9b7-c4cc-4fea-b2e0-4aeb0362ddde.jpeg)

Can anyone help?

**Comment by andresesfm** [comment]
That dialog is displayed by Google Play, I don't think we have control over it. Please report it to Google Play
